README.md

```diff
@@ -108,6 +108,15 @@ Now, add the following to your `Cargo.toml` to start building the libseccomp cra
 libseccomp = "0.2.3"
 ```
 
+### Building libseccomp with the `bundled` feature
+
+By enabling the bundled feature (see below), libseccomp-rs will automatically download, build, and link libseccomp for you. See the README for libseccomp-sys for more information.
+
+```toml
+[dependencies]
+libseccomp = { version = "0.2.3", features = ["bundled"] }
+```
+
```

Comment:
Cloud you fix this as follows? (We need to tell users that this feature **statically** links the libseccomp library by default)

### Building and statically linking the libseccomp library with the `bundled` feature

By enabling the bundled feature (see below), the libseccomp crate will automatically download,
build, and **statically** link the [libseccomp library ](https://github.com/seccomp/libseccomp) for you. 

```toml
[dependencies]
libseccomp = { version = "0.2.3", features = ["bundled"] }
```

The bundle feature downloads and builds the libseccomp sources according to the file [versions.rs](https://github.com/libseccomp-rs/libseccomp-rs/blob/main/libseccomp-sys/versions.rs)
by default if the `LIBSECCOMP_SRC_PATH` environment variable is not set. If you want to build
the libseccomp sources you already has on the system, set the `LIBSECCOMP_SRC_PATH`  to
specify the sources as follows.

```sh
$ export LIBSECCOMP_SRC_PATH="the path of the directory containing the libseccomp sources"
```

> **Note**: This feature links the libseccomp library statically by default.
> If you want to link the libseccomp library dynamically, you should not enable this feature.

Reply:
I've updated the README files, let me know what you think!